data/racoon/racoon.conf

```diff
@@ -0,0 +1,52 @@
+path include "/etc/racoon";
+path pre_shared_key "/etc/racoon/psk.txt";
+path certificate "/etc/racoon/cert";
+
+padding
+{
+    maximum_length 20;  # maximum padding length.
+    randomize off;      # enable randomize length.
+    strict_check off;   # enable strict check.
+    exclusive_tail off; # extract last one octet.
+}
+
+listen
+{
+    isakmp listenIP [500];
+}
+
+timer
+{
+    counter 5;      # maximum trying count to send.
+    interval 20 sec;    # maximum interval to resend.
+    persend 1;      # the number of packets per send.
+    phase1 30 sec;
+    phase2 15 sec;
+}
+
+remote RemoteIP
+{
+       exchange_mode main,aggressive;
+       doi ipsec_doi;
+       situation identity_only;
+       my_identifier asn1dn;
+       certificate_type x509 "vm01.cert" "vm01.key";
+       ca_type x509 "ca.cert";
```

Comment:
Please check the certificates name carefully. 
Make sure correct certificate is loaded.